frontend/src/metabase/dashboard/components/DashCard/DashCardActionButtons/DashCardActionsPanel.tsx

```diff
@@ -40,11 +44,11 @@ interface Props {
   ) => void;
   showClickBehaviorSidebar: () => void;
   onPreviewToggle: () => void;
-  onDashCardMenuClose: () => void;
-  onDashCardMenuOpen: () => void;
+  onLeftEdge: boolean;
+  onMouseDown: (event: React.MouseEvent) => void;
```

Comment:
Hmm, `React` is not imported.
Also, we use React things without `React.` prefix more often.